benchto-driver/src/main/java/io/trino/benchto/driver/execution/BenchmarkExecutionDriver.java

```diff
@@ -24,6 +24,7 @@
 import io.trino.benchto.driver.execution.BenchmarkExecutionResult.BenchmarkExecutionResultBuilder;
```

Comment:
when there are generated now?

Reply:
In [QueryExecutionDriver.java](https://github.com/trinodb/benchto/pull/49/files#diff-15760f0fc367513b1488223d0b9aff29153c3d76f91711ceca4af260bd7abeb8), right before executing the query.